configs/stdc/stdc-1446_pretrain_512x1024_80k_cityscapes.py

```diff
@@ -0,0 +1,6 @@
+_base_ = './stdc-1446_512x1024_80k_cityscapes.py'
+model = dict(
+    backbone=dict(
+        stdc_cfg=dict(
+            init_cfg=dict(
+                checkpoint='./pretrained/pretrained_stdc-1446.pth'))))
```

Comment:
```suggestion
                checkpoint='./pretrained/stdc-1446.pth'))))
```
Is `1446` official name?

Reply:
Yes, for example the name `813` and `1446` can be found [at original repo checkpoints link](https://drive.google.com/drive/folders/1wROFwRt8qWHD4jSo8Zu1gp1d6oYJ3ns1).